src/velero/chart/templates/uds-package.yaml

```
@@ -0,0 +1,45 @@
+apiVersion: uds.dev/v1alpha1
+kind: Package
+metadata:
+  name: velero
+  namespace: {{ .Release.Namespace }}
+spec:
+  network:
+    allow:
+      # Todo: wide open for now for pushing to s3
+      - direction: Egress
+        podLabels:
+          app.kubernetes.io/name: velero
+        remoteGenerated: Anywhere
+
+      - direction: Egress
+        podLabels:
+          batch.kubernetes.io/job-name: "velero-upgrade-crds"
+        remoteGenerated: Anywhere
```

Should this be KubeAPI rather than Anywhere? Would expect this job to hit kubeapi but not much else?

---

That's very fair. The networking rules were the thing I understood the least here. I'm sure I just copy/pasted this from another pkg 😆 

I could use a hand on the correct syntax though unless it's literally just replacing `Anywhere` -> `KubeAPI`.

---

Yep should be literally that! https://github.com/defenseunicorns/uds-core/blob/main/src/neuvector/chart/templates/uds-package.yaml#L25
```suggestion
      - direction: Egress
        podLabels:
          batch.kubernetes.io/job-name: "velero-upgrade-crds"
        remoteGenerated: KubeAPI
```
Should've added the suggestion on the first comment - just wasn't sure if there was something I was missing with other egress here.